README.md

```diff
@@ -0,0 +1,3 @@
+## Generalized-RPS-Game
+
+Final Project for CSC 222-1H0F SP23 at Laurel Ridge Community College. This is a generealized version of "Rock, Paper, Scissors".
```

Comment:
What more can you say here?

Reply:
Will add to the Readme the comments I had made for the 1.0 and 1.01 versions